modules/apps/27-interchain-accounts/controller/keeper/msg_server.go

```diff
@@ -49,7 +49,6 @@ func (s msgServer) SendTx(goCtx context.Context, msg *types.MsgSendTx) (*types.M
 		return nil, err
 	}
 
-	// explicitly passing nil as the argument is discarded as the channel capability is retrieved in SendTx.
 	absoluteTimeout := uint64(ctx.BlockTime().UnixNano()) + msg.RelativeTimeout
 	seq, err := s.sendTx(ctx, msg.ConnectionId, portID, msg.PacketData, absoluteTimeout)
```

Comment:
Maybe we could add something about the absolute timeout here instead:

```suggestion
        // the absolute timeout value is calculated using the controller chain block time + the relative timeout value
        // this assumes time synchrony to a certain degree between the controller and counterparty host chain
	absoluteTimeout := uint64(ctx.BlockTime().UnixNano()) + msg.RelativeTimeout
	seq, err := s.sendTx(ctx, msg.ConnectionId, portID, msg.PacketData, absoluteTimeout)
```